scan.go

```diff
@@ -207,62 +206,66 @@ func scan(pkgPatterns []string, scanDir string) ([]*unit.SourceUnit, error) {
 	var units []*unit.SourceUnit
 	for {
 		var pkg *build.Package
-		if err := dec.Decode(&pkg); err == io.EOF {
-			break
-		} else if err != nil {
+		if err := dec.Decode(&pkg); err != nil {
+			if err == io.EOF {
+				break
+			}
 			return nil, err
 		}
 
-		pv, pt := reflect.ValueOf(pkg).Elem(), reflect.TypeOf(*pkg)
+		// Collect all files
+		files := make([]string, 0)
```

Comment:
If you're making length 0, you might as well write this as `var files []string` which is simpler IMO.

I think @davecheney would agree. http://dave.cheney.net/2014/05/24/on-declaring-variables